Chocolatey.Cake.Recipe/Content/credentials.cake

```diff
@@ -19,6 +19,41 @@ public class DiscordCredentials
     public string UserName { get; private set; }
     public string AvatarUrl { get; private set; }
 
+    public static DiscordCredentials FetchCredentials(ICakeContext context)
+    {
+        var webHookUrl = context.EnvironmentVariable(Environment.DiscordWebHookUrlVariable);
+
+        if (string.IsNullOrEmpty(webHookUrl))
+        {
+            context.Warning("The required Web Hook URL for posting to Discord has not been provided.");
+            context.Warning("In future, this can be set using the {0} environment variable.", Environment.DiscordWebHookUrlVariable);
+
+            webHookUrl = context.Prompt("Enter Web Hook URL for posting to Discord:");
```

Comment:
So this suggestion is going to go for all of the `context.Prompt`s... We should provide a timeout... https://cakebuild.net/api/Cake.Common.IO/PromptAliases/58CD8679. I would suggest we set a global timeout and use it in all of them. I don't think it needs to be configurable at all, just something that can force the failure if it's run somewhere noninteractively.

Reply:
I have tested the latest package version for Cake.Prompt, and even there the timeout for the Prompt doesn't seem to be doing anything.

As a result, I have updated this PR with the suggestion that was made by Kim, to only attempt the usage of the prompt when running locally.